conf/messages.en

```diff
@@ -73,7 +73,13 @@ error.submissionError.link = start your claim again
 session_expired.title = For your security, this service has been reset
 session_expired.heading = For your security, this service has been reset
 session_expired.guidance = The details you have given have been deleted because you did not continue the service for 15 minutes.
+session_expired.signedOut = For your security, we signed you out
```

Comment:
I don't see use of this message anywhere.. please remove it if it is not using

Reply:
Removed.